erpnext/controllers/accounts_controller.py

```diff
@@ -237,6 +237,10 @@ def set_missing_item_details(self, for_validate=False):
 				document_type = "{} Item".format(self.doctype)
 				parent_dict.update({"document_type": document_type})
 
+                        # To fix the problems cause by renaming 'customer' to 'party_name- in selling docs
+                        if not parent_dict.get("customer") and parent_dict.get("party_name"):
```

Comment:
The field is only renamed for Quotation document. 

Also now you can quote to a customer as well as lead, so before settings party_name value directly to a customer, first you need to check whether the quotation is against customer or lead.

Reply:
Hi @saurabh6790 

   Thanks for the review, will do the updates based on your feedback.